docs/devGuide/githubActions/workflowSecurity.md

```diff
@@ -0,0 +1,100 @@
+{% set title = "GitHub Actions: Workflow Security" %}
+<span id="title" class="d-none">{{ title }}</span>
+
+<frontmatter>
+  title: "{{ title }}"
+  layout: devGuide.md
+  pageNav: default
+</frontmatter>
+
+# {{ title }}
+
+<div class="lead">
+
+GitHub Actions are a powerful tool for automating workflows. However, it is important to ensure that your workflows are secure to prevent malicious actors from exploiting them.
+
+</div>
+
+**Here are some best practices to follow when working on GitHub Actions workflows.**
```

Comment:
I would prefer if the div ends after this line as I feel like this bold small words looks a bit awkward